cmd/coder/main.go

```diff
@@ -15,6 +20,48 @@ import (
 func main() {
 	rand.Seed(time.Now().UnixMicro())
 
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	// Add a custom SIGQUIT handler that outputs to stderr and a well-known file
+	// in the home directory. This also prevents SIGQUITs from killing the CLI.
+	go func() {
+		sigs := make(chan os.Signal, 1)
+		signal.Notify(sigs, syscall.SIGQUIT)
```

Comment:
Will this work on Windows?

Reply:
This will do nothing on Windows

Reply:
I added a runtime.GOOS check so we get rid of the idle goroutine, but this would be harmless otherwise 👍